airflow/providers/trino/hooks/trino.py

```diff
@@ -145,6 +147,7 @@ def get_conn(self) -> Connection:
             # type: ignore[func-returns-value]
             isolation_level=self.get_isolation_level(),
             verify=_boolify(extra.get('verify', True)),
+            session_properties=ast.literal_eval(session_properties) if session_properties else None,
```

Comment:
Any reason you're using `ast.literal_eval` here instead of `json.loads`?

Reply:
I don't think it even make sense to deserialise `extra` because in Connection (airflow core) this field expected as JSON Object  or otherwise it parse an empty object

https://github.com/apache/airflow/blob/d370362d147ecbbcb699acea3aa33f0be5e24c16/airflow/models/connection.py#L396-L410

Reply:
@Taragolis @o-nikolas Thank you for your comments. I used `ast.literal_eval` thinking these values need to be a string but I was wrong. I will make changes 

```
{
  "catalog": "hive", 
  "protocol": "https", 
  "session_properties": {
    "hive.insert_existing_partitions_behavior": "OVERWRITE", 
    "scale_writers": true, 
    "task_writer_count": 1, 
    "writer_min_size": "32MB"
  }
}
```

This makes more sense.